src/Zynga/Framework/Lockable/Cache/V1/Driver/Caching.hh

```diff
@@ -130,6 +132,14 @@ class Caching extends FactoryDriverBase implements DriverInterface {
         }
         
         $lockRetryCount++;
+        
+        // We crossed max timeout, break early.
+        if( (microtime(true) - $startTime) > self::MAX_TIMEOUT_AMOUNT_SECONDS) {
+          return false;
+        }
+        
+        // Don't care about the first lock try. Only log an error for subsequent ones.
+        error_log('JEO pgData::lock storable=' . get_class($obj) . ' key=' . $lockKey . ' lockRetryCount=' . $lockRetryCount . ' obj=' . $obj->export()->asJSON());
```

Comment:
Double checking this error log is intended

Reply:
Yeah we want to add this